vcpkg/TOC.yml

```diff
@@ -45,7 +45,8 @@
       href: consume/install-locally-modified-package.md
     - name: Install a dependency from a Git-based registry
       href: consume/git-registries.md
-    - name: Authenticate to private Git registries
+    - name: Authenticate to private Git repositories
+      displayName: private registry, custom registry, GitHub

```

Comment:
This looks more like "tags"; is displayName really correct here?

Reply:
@vicroms pointed me to https://review.learn.microsoft.com/en-us/help/platform/navigation-toc-overview?branch=main out of band